ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -9,26 +9,26 @@ base_images:
     tag: golang-1.13
 build_root:
   image_stream_tag:
-    name: cincinnati-build-root
-    namespace: cincinnati-ci
-    tag: buildroot
+    name: centos
+    namespace: ci
+    tag: "7"
```

Comment:
ok, some progress.  Now I [get][1]:

```
2020/12/23 18:58:11 Build src failed, printing logs:
...
STEP 1: FROM image-registry.openshift-image-registry.svc:5000/ci-op-s3g02qis/pipeline@sha256:dc29e2bcceac52af0f01300402f5e756cc8c44a310867f6b94f5f7271d4f3fec
...
STEP 4: RUN umask 0002 && /clonerefs && find /go/src -type d -not -perm -0775 | xargs --max-procs 10 --max-args 100 --no-run-if-empty chmod g+xw
...
{"command":"git init","component":"clonerefs","error":"exec: \"git\": executable file not found in $PATH","file":"prow/pod-utils/clone/clone.go:61","func":"k8s.io/test-infra/prow/pod-utils/clone.Run.func2","level":"info","msg":"Ran command","output":"","severity":"info","time":"2020-12-23T18:58:05Z"}
```

So `src` is attempting to use the CentOS image:

```console
$ oc image info registry.build01.ci.openshift.org/ci-op-s3g02qis/pipeline@sha256:dc29e2bcceac52af0f01300402f5e756cc8c44a310867f6b94f5f7271d4f3fec | head -n3
Name:        registry.build01.ci.openshift.org/ci-op-s3g02qis/pipeline@sha256:dc29e2bcceac52af0f01300402f5e756cc8c44a310867f6b94f5f7271d4f3fec
Media Type:  application/vnd.docker.distribution.manifest.v2+json
Created:     2y ago
$ oc image info registry.ci.openshift.org/ci/centos:7 | head -n4
Name:        registry.ci.openshift.org/ci/centos:7
Digest:      sha256:dc29e2bcceac52af0f01300402f5e756cc8c44a310867f6b94f5f7271d4f3fec
Media Type:  application/vnd.docker.distribution.manifest.v2+json
Created:     2y ago
```

So, [as the docs point out][2], we need something with `git`.

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/14518/rehearse-14518-pull-ci-openshift-cincinnati-master-images/1341820303646396416#1:build-log.txt%3A19
[2]: https://docs.ci.openshift.org/docs/architecture/ci-operator/#build-root-image

Reply:
Pivoted to a Go root with 52fdbbd -> d06a535.  Would be nice if there was a default with `git`, so we could leave `build_root` unset and just have ci-tools pick whatever the going default was for "an image with `git`".  Or if it would allow us to just drop `build_root` entirely, because I don't think we actually consume any of those images...